docs-sources/includes/_Configuration.md

```diff
@@ -3,11 +3,12 @@
 ## Format
 
 The configuration for your game is allowed to exist in either the
```

Comment:
The word `either` implies 2 I believe, so this sentence maybe deserves rephrasing.